[2025-05-19T18:03:29.616+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-19T18:03:29.665+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_task_car.load_to_postgres manual__2025-05-19T18:03:21.887264+00:00 [queued]>
[2025-05-19T18:03:29.687+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_task_car.load_to_postgres manual__2025-05-19T18:03:21.887264+00:00 [queued]>
[2025-05-19T18:03:29.688+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-05-19T18:03:29.733+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-05-19 18:03:21.887264+00:00
[2025-05-19T18:03:29.754+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1052) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-19T18:03:29.758+0000] {standard_task_runner.py:72} INFO - Started process 1054 to run task
[2025-05-19T18:03:29.756+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_task_car', 'load_to_postgres', 'manual__2025-05-19T18:03:21.887264+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/test_car.py', '--cfg-path', '/tmp/tmp3t7ga4c4']
[2025-05-19T18:03:29.761+0000] {standard_task_runner.py:105} INFO - Job 21: Subtask load_to_postgres
[2025-05-19T18:03:29.862+0000] {task_command.py:467} INFO - Running <TaskInstance: test_task_car.load_to_postgres manual__2025-05-19T18:03:21.887264+00:00 [running]> on host db4eb1a3da04
[2025-05-19T18:03:30.073+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_task_car' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T18:03:21.887264+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-19T18:03:21.887264+00:00'
[2025-05-19T18:03:30.077+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-19T18:03:30.152+0000] {test_car.py:76} INFO - >>> [START] load_to_postgres
[2025-05-19T18:03:30.278+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2025-05-19T18:03:30.300+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'Toyota', 'model': 'Camry', 'engine_volume': 2.5, 'manufacture_year': 2020, 'price_foreign': 25000, 'price_rub': 2250000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.301+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'Honda', 'model': 'Accord', 'engine_volume': 1.5, 'manufacture_year': 2021, 'price_foreign': 22000, 'price_rub': 1980000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.303+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'BMW', 'model': 'X5', 'engine_volume': 3.0, 'manufacture_year': 2019, 'price_foreign': 45000, 'price_rub': 4050000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.304+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'Mercedes', 'model': 'E-Class', 'engine_volume': 2.0, 'manufacture_year': 2022, 'price_foreign': 40000, 'price_rub': 3600000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.304+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'Audi', 'model': 'A6', 'engine_volume': 2.0, 'manufacture_year': 2021, 'price_foreign': 38000, 'price_rub': 3420000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.305+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'Kia', 'model': 'Optima', 'engine_volume': 2.4, 'manufacture_year': 2020, 'price_foreign': 18000, 'price_rub': 1620000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.306+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'Hyundai', 'model': 'Solaris', 'engine_volume': 1.6, 'manufacture_year': 2021, 'price_foreign': 15000, 'price_rub': 1350000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.307+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'Volkswagen', 'model': 'Tiguan', 'engine_volume': 2.0, 'manufacture_year': 2022, 'price_foreign': 30000, 'price_rub': 2700000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.308+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'Lexus', 'model': 'RX', 'engine_volume': 3.5, 'manufacture_year': 2021, 'price_foreign': 50000, 'price_rub': 4500000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.309+0000] {test_car.py:126} ERROR - Error inserting record: {'brand': 'Ford', 'model': 'Focus', 'engine_volume': 1.5, 'manufacture_year': 2020, 'price_foreign': 16000, 'price_rub': 1440000, 'file_actual_date': '2025-05-18'}. Error: 'processing_date'
[2025-05-19T18:03:30.310+0000] {test_car.py:130} INFO - Successfully loaded 10 records to PostgreSQL
[2025-05-19T18:03:30.311+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-19T18:03:30.342+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-19T18:03:30.344+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_task_car, task_id=load_to_postgres, run_id=manual__2025-05-19T18:03:21.887264+00:00, execution_date=20250519T180321, start_date=20250519T180329, end_date=20250519T180330
[2025-05-19T18:03:30.463+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-19T18:03:30.509+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-19T18:03:30.512+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
